web/content/index.adoc

```diff
@@ -0,0 +1,14 @@
+---
+title: Home
+---
+
+== Foreman and Katello Documentation
+
+* link:/release/nightly/[Nightly build]
+* link:/release/3.1/[3.1]
+* link:/release/3.0/[3.0 (stable)]
+* link:/release/2.5/[2.5 (supported)]
+
+Work in progress documentation:
```

Comment:
I'd still like to hold off on giving the impression that it's ready until we actually agree on that. For example, right now 2.5 is actually End of Life upstream. 3.1 is the stable version. There's probably more things.

Can we keep the "work in progress" label for now, merge this and iterate?

Reply:
> Can we keep the "work in progress" label for now

What you mean by this? You want me to delete this line, or put some note?

Reply:
I corrected the list above, it was incorrect due to how long I was working on this patch.